### PR TITLE
add universe builder to ddx dbt models

### PR DESCRIPTION
**Pull request by KelseyNewman147**
This PR adds the dbt for the `universe-builder` and `universe-builder-raw` streams from DDx as part of [DE-772](https://techallies.atlassian.net/browse/DE-772)

[DE-772]: https://techallies.atlassian.net/browse/DE-772?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by kanelouise — DISMISSED** [review]
Nice! LGTM!

File: dbt-cta/ddx/models/1_cta_incremental/universe_builder_raw_base.sql [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{{ config(
+    partition_by = {"field": "datetime_pulled", "data_type": "timestamp", "granularity": "day"},
+    unique_key = "_cta_hash_id",
+    tags = [ "universe-builder-raw" ],
+    persist_docs = {"columns": true}
```
[the remark this answers is not in the document]
Comment (KelseyNewman147) [reply dbt-cta/ddx/models/1_cta_incremental/universe_builder_raw_base.sql:5]: It doesn't look like there are any column descriptions. I'll admit, I copied this from the `survey_response_base.sql`. Is this supposed to copy it from the model?

File: dbt-cta/ddx/models/sources.yml [hunk 2]
```diff
@@ -4,4 +4,6 @@ sources:
   database: "{{ env_var('CTA_PROJECT_ID') }}"
   schema: "{{ env_var('CTA_DATASET_ID') }}"
   tables:
-    - name: _survey_response_raw
\ No newline at end of file
+    - name: _survey_response_raw
+    - name: _universe_builder_raw
+    - name: _universe_builder_raw_raw
```
[the remark this answers is not in the document]
Comment (KelseyNewman147) [reply dbt-cta/ddx/models/sources.yml:9]: haha I know, it hurts me too but I had to do it

File: dbt-cta/ddx/models/1_cta_incremental/universe_builder_raw_base.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (KelseyNewman147) [reply dbt-cta/ddx/models/1_cta_incremental/universe_builder_raw_base.sql]: Yeah I can't get this to work 😞

[the remark this answers is not in the document]
Comment (KelseyNewman147) [reply dbt-cta/ddx/models/1_cta_incremental/universe_builder_raw_base.sql]: haha yeah, it seems like it should work? we're doing what the [docs](https://docs.getdbt.com/reference/resource-configs/persist_docs) say to do 🤷‍♀️ I'm going to leave the config and may tinker with it a bit more later.

**Pull request merged**